"""
Introdução ao try/except
try -> tentar executar o código
except -> ocorreu algum erroa ao tentar executar
"""
numero  = input('Vou dobrar o número que vc digitar: ')
numeor = ...

numero_float = float(numero)
print(f'O dobro de {numero_float} é {numero_float * 2:.2f}')